### ISSUE TITLE
Support for Composer 1 is deprecated and some packages will not be available.

### ISSUE THREAD

**Issue by kenjis** [FriendsOfPHP/security-advisories#667]
```
$ composer update
Loading composer repositories with package information
Warning from https://repo.packagist.org: Support for Composer 1 is deprecated and some packages will not be available. You should upgrade to Composer 2. See https://blog.packagist.com/deprecating-composer-1-support/
```

**Comment by derrabus** [comment]
Okay, what is your question?

**Comment by kenjis** [comment]
I don't know why, but this repository depends on Composer 1.

https://github.com/FriendsOfPHP/security-advisories/blob/87b2af8f33ce764f959ce67fe322391b2c45ad06/composer.json#L5

**Comment by xabbuh** [comment]
All the dependencies haven't been updated for a while probably because the PHP script is only used in the pipeline when verifying committed vulnerabilities.

**Comment by stof** [comment]
Looks like you get this error when running composer, not when running the validator. So make sure your `composer update` uses Composer 2.

regarding the dev dependency, I'm updating it in #668 but that should be irrelevant to this issue.

**Comment by kenjis** [comment]
Thank you. #668 fixed this issue and #666.
